packages/calypso-e2e/src/secrets/secrets-manager.ts

```diff
@@ -151,7 +151,12 @@ export class SecretsManager {
 				totpUserInboxId: 'FAKE_VALUE',
 			},
 			testAccounts: {
-				defaultUser: { ...fakeAccount },
+				defaultUser: {
+					...fakeAccount,
+					userID: 0,
+					testSites: { primary: { id: 0, url: 'FAKE_VALUE' } },
+					email: 'FAKE_VALUE',
+				},
```

Comment:
`defaultUser` has had the `userID` or `email` attributes for some time, but it was not reflected in the type-checker here.